### PR TITLE
Network Concept added

### PR DESCRIPTION
**Pull request by datenbrei**
@roschaefer @appinteractive , I added the missing concept part of the developer documentation. 

**Background Info**: I was talking with Dennis before to clarify the headlines and the features. This had just been partitially done and further work on it will follow.

### CODE REVIEW

File: concept/concept.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+- Filter by Category (Health and Wellbeing, Global Peace & Non-Violance, ...) 
+- Filter by Mood (Funny, Happy, Surprised, Cry, Angry, ...)
+- Filter by Source (Connections, Following, Individuals, Non-Profits, ...)
+- Filter by Posts & Tools (Post, Events, CanDos, ...)
+- Filter by Format Type (Text, Pictures, Video, ...)
+- Extended Filter (Continent, Country, Language, ...) 
+- Sort Posts by Date
+- Sort Posts by Shouts
+- Sort Posts by most discussed
+- Sort Posts by most Emojis
```
Comment (appinteractive) [inline concept/concept.md]: Most different emojis or voting count in general?

File: concept/concept.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+### Blacklist
+
+[Video](https://www.youtube.com/watch?v=-uDvvmN8hLQ)
+
+- Blacklist Users
+- Blacklist Subjects
```
Comment (appinteractive) [inline concept/concept.md]: What is a subject in that case?

File: concept/concept.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+- Editing Why - why should you do this
+- Editing Usefulness - what is it good for
+
+### Versus (interaction on existing Post)
+
+- Create / edit / delete Versus
+
+### Jobs
+
+- Create, edit and delete Jobs
```
Comment (appinteractive) [inline concept/concept.md]: Jobs can be created by everyone?

File: concept/concept.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+- Rate up Arguments
+- Add Tags
+- Attach Media
+
+### Votes
+
+- Create Votes (Surveys)
+- Add Title and Description
+- Let Users vote
+- Set Vote Type (Yes/No or Poll)
```
Comment (appinteractive) [inline concept/concept.md]: Why a type? Just create at least two options and name then yes no or do how ever you wish, type is not needed from my perspective.

File: concept/concept.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+- Create Votes (Surveys)
+- Add Title and Description
+- Add Tags
+- Let Users vote for Best Item
+- Set Settings (allow Uploads, allow Links, ...)
+
+### Events
+
+- Create Events
+- Add Title and Description
```
Comment (appinteractive) [inline concept/concept.md]: Joe about date and location? 😅

File: concept/concept.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+
+### More Info
+
+Shows autmatically releated information for existing post.
+
+- Show related Posts
+- Show Pros and Cons
+- Show Bestlist
+- Show Votes
+- Link to responding Chatroom
```
Comment (appinteractive) [inline concept/concept.md]: Responding or corresponding?

File: concept/concept.md (hunk 4 above)
Comment (datenbrei) [reply concept/concept.md]: Agreed. Will change this and re-commit/push to branch

**Comment by datenbrei** [comment]
This is what "further work on it will follow" means - it is not yet complete :smile:  . Most details had been written down / extracted  from clickdummies and some will need further investigation.

**Comment by datenbrei** [comment]
It is plaaned for each of these details "headline" to write a user story with more clear details and explanations. These will be referenced.

File: concept/concept.md [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+
+- **Knowledge**: Read articles about interesting topics and find related posts in the **More Info** tab or by **Filtering** based on **Categories** and **Tagging** or by using the **Fulltext Search**.
+
+- **Action**: Don't just read about how to make the world a better place, but come into **Action** by following provided suggestions on the **Action** tab provided by other people or **Organisations**.
+
+## Features
+
+The following features will be implemented. This gets done in three steps: 
+
+1. First we will implement a basic feature set and provide an test system for to test the basic network functionality.
```
Comment (roschaefer) [inline concept/concept.md]:
```suggestion
1. First we will implement a basic feature set and provide a test system to test the basic network functionality. 
```

File: concept/concept.md [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+
+- **Action**: Don't just read about how to make the world a better place, but come into **Action** by following provided suggestions on the **Action** tab provided by other people or **Organisations**.
+
+## Features
+
+The following features will be implemented. This gets done in three steps: 
+
+1. First we will implement a basic feature set and provide an test system for to test the basic network functionality. 
+
+2. In a second step we will go live into the public with an advanced feature set which includes also all the functionality in code and organisation that will be needed to drive a bigger public social network.
```
Comment (roschaefer) [inline concept/concept.md]:
```suggestion
2. In a second step we will make our prototype publicly available with an advanced feature set including the technology and organizational structure to drive a bigger public social network.
```

File: concept/concept.md [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+
+3. In a third step all the remaining Features will be implemented to build the full product.
```
Comment (roschaefer) [inline concept/concept.md]:
```suggestion
3. In a third step all the remaining features will be implemented to build the full product. 
```
Will there ever be a "finished" product? :thinking: :wink:

File: concept/concept.md [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+- Creating Comments 
+- Deleting Comments
+- Editing Comments
+- Upvote comments of others
+
+### Article List
+
+- Show Posts by Tiles
+- Show Posts as List
+- Filter by Category (Health and Wellbeing, Global Peace & Non-Violance, ...)
```
Comment (roschaefer) [inline concept/concept.md]:
```suggestion
- Filter by Category (Health and Wellbeing, Global Peace & Non-Violence, ...) 
```

File: concept/concept.md [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+- Sort Posts by most Emojis
+
+### Blacklist
+
+[Video](https://www.youtube.com/watch?v=-uDvvmN8hLQ)
+
+- Blacklist Users
+- Blacklist Subjects
+- Blacklist Tags
+- Switch on/off Adult Content
```
Comment (roschaefer) [inline concept/concept.md]: Mastodon and Peertube classify content (e.g. `nsfw` == not safe for work). Classification is a feature on its own.

**Comment by roschaefer** [comment]
@datenbrei You said you want to change things, no? In that case I will wait before hitting the merge button. Otherwise just tell me and we get this PR in right now.

File: concept/concept.md (hunk 1 above)
Comment (datenbrei) [reply concept/concept.md]: Clarified and changed in pushed in branch network-concept

File: concept/concept.md (hunk 2 above)
Comment (datenbrei) [reply concept/concept.md]: Clarified and changed in updated branch network-concept

File: concept/concept.md (hunk 3 above)
Comment (datenbrei) [reply concept/concept.md]: Yes. Clarified and changed in updated branch network-concept

File: concept/concept.md (hunk 4 above)
Comment (datenbrei) [reply concept/concept.md]: Ripped out vote types after clarification with Dennis. See in updated branch network-concept

File: concept/concept.md (hunk 5 above)
Comment (datenbrei) [reply concept/concept.md]: did add these in updated branch network-concept

File: concept/concept.md (hunk 6 above)
Comment (datenbrei) [reply concept/concept.md]: The latter. See updated branch network-concept

File: concept/concept.md (hunk 9 above)
Comment (datenbrei) [reply concept/concept.md]: Corrected in updated branch network-concept

File: concept/concept.md (hunk 11 above)
Comment (datenbrei) [reply concept/concept.md]: Yes, indeed.

**Comment by datenbrei** [comment]
Hi guys! These had been my first steps with git. well, they say "The first time hurts.". :-)

After doing all the stuff I compiled a short tutorial for myself to not forget the flow of commands. Maybe it is of use for others too. Thanks for commenting, Greg, and thanks for your help and introducing me, Robert.

## Git short introduction for changing code

**Clone** an existing repository for later applying own changes to it.
~~~
git clone https://github.com/Owner/Project.git
~~~
**Create new branch** for own features to be written or changes to be applied:
~~~
git checkout -b branch-name
~~~
**Edit** the file(s) and stash (add) the changed files again:
~~~
git add example.txt 
~~~
Now **commit** the added files to the local repository:
~~~
git commit -m "Short description of applied changes"
~~~
Before pushing up own changes fetch and merge (**pull**) in changes from remote repository:
~~~
git pull origin branch-name
~~~
Now **push local changes** to remote repository
~~~
git push origin branch-name
~~~

**Pull request merged**